CRUD.c

```diff
@@ -0,0 +1,186 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define FILENAME "users.txt"
+
+// User structure

```

Comment:
Remove all redundant comments from the code

Reply:
👍